src/main/java/engineer/skyouo/plugins/naturerevive/config/ReadonlyConfig.java

```diff
@@ -136,7 +142,7 @@ public ReadonlyConfig() {
 
     private void updateConfigurations(int version) {
         switch (version) {
-            case 1:
+            case 2:
```

Comment:
此處建議改為

```java
case 1:
    configuration.set("griefprevention-strict-check", false);
    configuration.setComments("griefprevention-strict-check", Arrays.asList("是否啟用 再生含有GP領地的區塊，但是不再生GP領地範圍內的方塊 功能",
                    "Whether to enable the experimental function that if the expired chunk has GriefPrevention in it, put all blocks in GriefPrevention to new chunk instead of skipping chunk."));
```

以便與上個 `CONFIG_VERSION` 兼容.

Reply:
以新增